cookbook/security/remember_me.rst

```diff
@@ -171,21 +171,25 @@ In the following example, the action is only allowed if the user has the
         // ...
     }
 
-You can also choose to install and use the optional JMSSecurityExtraBundle_,
-which can secure your controller using annotations:
+If your application is based on the Symfony Standard Edition, you can also secure
+your controller using annotations:
 
 .. code-block:: php
 
-    use JMS\SecurityExtraBundle\Annotation\Secure;
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 
     /**
-     * @Secure(roles="IS_AUTHENTICATED_FULLY")
+     * @Security("has_role('IS_AUTHENTICATED_FULLY')")
      */
     public function editAction($name)
     {
         // ...
     }
 
+.. versionadded:: 2.4
+    The ``@Security`` annotation was introduced in SensioFrameworkExtraBundle 3.0,
+    which can only be used with Symfony 2.4 or later.
```

Comment:
As I said [here](https://github.com/symfony/symfony-docs/pull/4735#issuecomment-68540224), we shouldn't over-complicate things here. Even the bundle's readme just states "The minimum version of Symfony2 for this workflow is 2.4.0.".